bin/index.ts

```diff
@@ -183,13 +183,13 @@ async function getCommandLineOptions(argDefinitions: Array<CommandLineDefinition
         version
     }
 }
+
 // The rust-minidump-stackwalker symbol lookup implementation removes some extensions from the sym file name for symbol lookups.
 // This is a bit of a mystery and is subject to change when we learn more about how it works.
 // For now, remove any non .so extension in the sym file's base name to satisfy the minidump-stackwalker symbol lookup.
 function getSymFileBaseName(file: string): string {
     const linuxSoExtensionPattern = /\.so\.?.*$/gm;
-    const fileNoExt = file.split('.')[0];
```

Comment:
This was failing in the SEA because under these conditions the paths started with . and .., and this would also break if there was a . anywhere else in the path. We did this because we wanted to turn bugsplat.app.dSYM to bugsplat.sym. We now just recursively remove all parts of the extension using built in node functions.